core/domain/feedback_jobs_one_off.py

```diff
@@ -182,3 +184,95 @@ def map(item):
     @staticmethod
     def reduce(key, value):
         yield (key, len(value))
+
+
+class GeneralizeFeedbackThreadMigrationOneOffJob(
+        jobs.BaseMapReduceOneOffJobManager):
+    """One off job to populate two new fields for instances of Feedback thread
+    model.
+    """
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [feedback_models.FeedbackThreadModel]
+
+    @staticmethod
+    def map(thread):
+        thread.entity_type = 'exploration'
+        thread.entity_id = thread.exploration_id
+        thread.put()
+
+    @staticmethod
+    def reduce(thread):
+        pass
+
+
+class FeedbackThreadIdMigrationOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """One-off job for migrating instances of feedback thread model and related
+    models to have thread ID of the for entity_type.entity_id.random_str.
```

Comment:
of the for? I think we should revisit this comment

Reply:
possibly "for" --> "form"

Reply:
Done